explore.ts

```diff
@@ -0,0 +1,58 @@
+import { Directions } from "./contants"
+import Plateau from "./plateau";
+import validate from "./validate";
+
+const explore = (currentX: number, currentY: number, currentDirection: Directions, instruction: string, plateau: Plateau) => {
```

Comment:
Better to have coordinates as a class/interface in itself. That way, you will be able to abstract a lot of logic within it.

Reply:
True.